lib/active_record/turntable/active_record_ext/fixtures.rb

```diff
@@ -105,7 +105,30 @@ def setup_fixtures(config = ActiveRecord::Base)
           end
         end
 
-        if ActiveRecord::Turntable::Util.ar51_or_later?
+        if ActiveRecord::Turntable::Util.ar71_or_later?
+          @connection_subscriber = ActiveSupport::Notifications.subscribe("!connection.active_record") do |_, _, _, _, payload|
+            connection_name = payload[:connection_name] if payload.key?(:connection_name)
+            shard = payload[:shard] if payload.key?(:shard)
+  
+            if connection_name
+              begin
+                connection = ActiveRecord::Base.connection_handler.retrieve_connection(connection_name, shard: shard)
+              rescue ConnectionNotEstablished
+                connection = nil
+              end
+  
+              if connection
+                setup_shared_connection_pool
+  
+                if !@fixture_connections.include?(connection)
```

Comment:
@haraki-tbn 
マージ後のコメントですみません。🙇
`exclude?`を使用した方がわかりやすいと感じました。
```suggestion
                if @fixture_connections.exclude?(connection)
```

※ 改めて修正をする必要はないです。🙇

Reply:
ありがとうございます 🙇 
ここらへんのコードは元の`active_record`からコピって上書きしており、差分をだしたくないのでこのままにさせてもらいますね 🙏 
https://github.com/rails/rails/blob/v7.1.5.1/activerecord/lib/active_record/test_fixtures.rb#L169-L172